### DISCUSSION TITLE
Added member, and he's stuck in "Pending"

### DISCUSSION THREAD

**Discussion by WesBanco7212** [community/community#106461] [Enterprise] [answered]
I added a member by sending an email invite. The member accepted the invite, however, when I go to the administration, it still shows him as pending. Not sure what else I need to do.

**Comment by WesBanco7212** [comment, marked as answer]
This has been resolved, canceled invite by email and recreated by inviting user ID. After that they were able to accept and it appears that all is normal.

**Comment by Gelarshid** [comment]
I sent an invitation, but even though the other party has confirmed it, it is still in the Pending Invite status. Please check.

**Reply by vladimirsvicevicsrb** [reply]
Any update on this? Having same issue
